src/main/java/org/zalando/nakadi/config/SecurityConfiguration.java

```diff
@@ -60,6 +60,7 @@ public void configure(final HttpSecurity http) throws Exception {
                     .antMatchers(POST, "/event-types/**").access(hasScope(eventTypeWriteScope))
                     .antMatchers(PUT, "/event-types/**").access(hasScope(eventTypeWriteScope))
                     .antMatchers(POST, "/subscriptions/**").access(hasScope(eventStreamReadScope))
+                    .antMatchers(GET, "/subscriptions/**").access(hasScope(eventStreamReadScope))
```

Comment:
Yes, we actually have some other things here which can be replaced by more general mask. But I would prefer not to do it because currently each line corresponds to an endpoint and I think it makes things more clear and easy.